### DISCUSSION TITLE
Change/update algorithm for thumb generation. **Especially** for Phoronix.

### DISCUSSION THREAD

**Discussion by p-kraszewski** [dailydotdev/daily#1112] [Feature Requests]
Please, change/update algorithm for thumb generation. **Especially** for Phoronix.
Michael's face saying _"I know what you've written in 1997 and the code is disgusting"_ sends shivers down my spine each time I see it...

**Comment by idoshamun** [comment]
Hi @p-kraszewski,

We have many team members triggered by these images as well. Unfortunately, there's no algorithm or anything we just take the social image provided by the site.
I hope we can fix this issue in the future
